pkg/auth/azure.go

```diff
@@ -152,7 +152,13 @@ func HttpAuthMiddleWare(resp http.ResponseWriter, req *http.Request, jwks *keyfu
 		}
 	}
 
-	err := ValidateToken(token, jwks, clientId, tenantId)
+	claims, err := ValidateToken(token, jwks, clientId, tenantId)
+
+	if _, ok := claims["tid"]; ok {
```

Comment:
tid will be present whether or not its an access token. check !appid instead.

appid is given if access token

https://github.com/freiheit-com/kuberpult/commit/14251afe7e133ad5c61549ac3cc44b48ec45f3a0

Reply:
Fixed, as now the `aud` claim is verified since it is used to check the App ID, according to https://docs.microsoft.com/en-us/azure/active-directory/develop/id-tokens#payload-claims